Makefile

```diff
@@ -202,8 +202,12 @@ lint_quick:
 	@echo "Quick linting on ONLY changed/new files"
 	@$(FIND_MOD_NEW) | xargs -r python2 lint.py
 
-# Disable import error
-PYLINT_DISABLE := F0401 duplicate-code
+# Disable all the following pylint messages:
+PYLINT_DISABLE := \
+	missing-module-docstring missing-class-docstring missing-function-docstring \
+	import-error invalid-name redefined-outer-name unused-argument \
+	too-few-public-methods duplicate-code no-self-use
```

Comment:
I'd appreciate if some of these could be disabled only for MPXE (the `missing-X-docstring` messages I've found have been really good for improving documentation in babydriver, and `invalid-name` probably shouldn't be disabled outside MPXE). The rest look good to disable everywhere though.

Also, thanks for expanding `F0401` into the more readable `import-error` :)